packages/renderer/src/lib/pod/DeployPodToKube.svelte

```diff
@@ -35,6 +35,7 @@ let deployUsingServices = true;
 let deployUsingRoutes = true;
 let deployUsingRestrictedSecurityContext = false;
 let createdPod: V1Pod | undefined = undefined;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
issue: the type should not be any

Reply:
unknown, then checking if some fields are here and then the optional Types being in the type
if (field in myObject) {
 ....
}